lib/ancestry/instance_methods.rb

```diff
@@ -76,7 +76,7 @@ def touch_ancestors_callback
         if self.ancestry_base_class.touch_ancestors
 
           # Touch each of the old *and* new ancestors
-          self.class.where(id: (ancestor_ids + ancestor_ids_was).uniq).each do |ancestor|
+          unscoped_current_and_previous_ancestors.each do |ancestor|
```

Comment:
Maybe:

```ruby
ancestry_base_class.unscoped.where(id: (ancestor_ids + ancestor_ids_was)).each do |ancestor|
```

Don't think the `uniq` or the `self` is needed, but I haven't researched.

Think the separate method may be a bit much unless you see it used elsewhere.

Either way, I'd prefer the single line version